components/emoji_picker/components/emoji_picker_item.jsx

```diff
@@ -10,30 +10,33 @@ export default class EmojiPickerItem extends React.PureComponent {
     static propTypes = {
         emoji: PropTypes.object.isRequired,
         onItemOver: PropTypes.func.isRequired,
-        onItemOut: PropTypes.func.isRequired,
         onItemClick: PropTypes.func.isRequired,
-        onItemUnmount: PropTypes.func.isRequired,
-        category: PropTypes.string.isRequired
-    }
+        category: PropTypes.string.isRequired,
+        isSelected: PropTypes.bool,
+        categoryIndex: PropTypes.number.isRequired,
+        emojiIndex: PropTypes.number.isRequired,
+        containerTop: PropTypes.number.isRequired,
+        containerBottom: PropTypes.number.isRequired
+    };
 
     constructor(props) {
         super(props);
-
         this.handleMouseOver = this.handleMouseOver.bind(this);
-        this.handleMouseOut = this.handleMouseOut.bind(this);
         this.handleClick = this.handleClick.bind(this);
     }
-
-    componentWillUnmount() {
-        this.props.onItemUnmount(this.props.emoji);
+    componentWillReceiveProps(nextProps) {
+        if (!this.props.isSelected && nextProps.isSelected) {
+            const offsetTop = this.emojiItem.offsetTop;
+            const height = this.emojiItem.offsetHeight * 1.2;
```

Comment:
Is the `* 1.2` to correct for spacing? Could we throw it into a `const` at the top of the file and use a descriptive name for it, that would make it a little more clear what it's used for

Reply:
Maybe we should look at the actual padding/margins on the component to get the exact height from the CSS. Using a multiplier seems inaccurate, and it'll break if we ever change the styling

Reply:
Done

Reply:
This was confusing. I updated the code and rewrote to be more explicit in what it does